src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -471,8 +471,13 @@ private void populateDirectoryList() {
             mListView.setOnItemClickListener(this);
         }
     }
-    
-    private void startSyncFolderOperation(OCFile folder) {
+
+    @Override
+    public void onSavedCertificate() {
+        startSyncFolderOperation(getCurrentDir(), false);
+    }
+
+    private void startSyncFolderOperation(OCFile folder, boolean ignoreETag) {
```

Comment:
Is the new parameter needed? All the calls I can see pass ```false```. Is there any change we need to call the method with ```ignoreEtag == true``` in a foreseeable future?